Rubberduck.VBEditor.VBA/SafeComWrappers/VB/CodeModule.cs

```diff
@@ -122,29 +124,48 @@ public void AddFromFile(string path)
 
         public void InsertLines(int line, string content)
         {
-            if (IsWrappingNullReference) return; 
-            Target.InsertLines(line, content);
+            if (IsWrappingNullReference) return;
+            try
+            {
+                Target.InsertLines(line, content);
+
+            }
+            catch (Exception e)
+            {
+                // "too many line continuations" is one possible cause for a COMException here.
+                Debug.Assert(false, e.ToString());
+            }
         }
 
         public void DeleteLines(int startLine, int count = 1)
         {
             if (IsWrappingNullReference) return;
             if (Target.CountOfLines > 0)
             {
-                Target.DeleteLines(startLine, count);
+
+                try
+                {
+                    Target.DeleteLines(startLine, count);
+
+                }
+                catch (Exception e)
+                {
+                    // "too many line continuations" is one possible cause for a COMException here.
```

Comment:
huh? I'll assume that this is a copy-paste mishap. We're deleting lines from originally valid code right? How can we get too many line continuations if we didn't insert lines?

Reply:
Wrong assumption: if we're deleting lines from illegal code with too many line continuations, this method throws if the result is still too many continuated lines.